File: car-lease-service/src/main/kotlin/org/jesperancinha/car/lease/dto/CustomerDto.java
package org.jesperancinha.car.lease.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class CustomerDto {
    private Long id;
    private String name;
    private String street;
    private Long houseNumber;
    private String zipCode;
    private String place;
    private String email;
    private String phoneNumber;
}

File: car-lease/car-lease-service/src/main/kotlin/org/jesperancinha/car/lease/dto/CustomerDto.kt
package org.jesperancinha.car.lease.dto

import com.fasterxml.jackson.annotation.JsonProperty

data class CustomerDto(
    @param:JsonProperty("id")
    val id: Long? = null,
    @param:JsonProperty("name")
    val name: String? = null,
    @param:JsonProperty("street")
    val street: String? = null,
    @param:JsonProperty("houseNumber")
    val houseNumber: Long? = null,
    @param:JsonProperty("zipCode")
    val zipCode: String? = null,
    @param:JsonProperty("place")
    val place: String? = null,
    @param:JsonProperty("email")
    val email: String? = null,
    @param:JsonProperty("phoneNumber")
    val phoneNumber: String? = null
)